app/models/request.rb

```diff
@@ -0,0 +1,6 @@
+class Request < ApplicationRecord
+  belongs_to :book
+  belongs_to :user
+  enum status: [:pending, :denied, :approved]
```

Comment:
The `enum` goes before the `belongs_to`. 
There's a style guide, you can check it here https://github.com/bbatsov/rails-style-guide#keep-ar-defaults